egcg_core/config.py

```diff
@@ -8,20 +8,23 @@ class Configuration:
     config_file = None
     content = {}
 
-    def __init__(self, *search_path):
+    def __init__(self, *search_path, env_var=None):
         if search_path:
-            self.load_config_file(self._find_config_file(search_path))
+            self.load_config_file(*search_path, env_var=env_var)
+            self._select_env()
 
     @staticmethod
     def _find_config_file(search_path):
         for p in search_path:
             if p and isfile(p):
                 return p
 
-    def load_config_file(self, *search_path):
+    def load_config_file(self, *search_path, env_var=None):
+        self.env_var = env_var
```

Comment:
We should initialise `self.env_var` in \_\_init\_\_, as well as `self.config_file` and `self.content`